ansible_collections/arista/avd/roles/eos_designs/docs/Input Variables.md

```diff
@@ -0,0 +1,4458 @@
+---
+search:
+  boost: 2
```

Comment:
If you rebase from latest devel this file should not be there any longer. Otherwise try to delete it locally and rerun pre-commit.